<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_dialog</class>
 <widget class="QDialog" name="new_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="toggle_btn1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgray; border-radius: 25px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>    OFF</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="toggle_btn2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgray; border-radius: 25px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>    OFF</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1_btn">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>45</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2_btn">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>115</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
